blog/_posts/2017-08-09-two-girls-meet-in-shanghai.md

```diff
@@ -0,0 +1,57 @@
+---
+title: So, two girls meet in Shanghai…?
+layout: post
+created_at: Wednesday August 09 2017
+permalink: blog/2017-08-09-two-girls-meet-in-shanghai
+current: blog
+author: BansheeBandits
+twitter: BansheeBandits
+categories:
+- student-posts
+---
+
+## Who are we?! How did we come together?! 
```

Comment:
Hi, thanks for the PR. :) Can you please use one symbol in all titles? I think 'question mark' should be fine in all titles.